app/components/dynamic-content/dynamic-content-form/dynamic-content-form-field/dynamic-content-form-field-boolean/dynamic-content-form-field-boolean.html

```diff
@@ -0,0 +1,13 @@
+<dynamic-content-form-field-input-label
+    name="{{ name }}"
+    schema="schema">
+</dynamic-content-form-field-input-label>
+<input
+    name="{{ name }}"
+    ng-model="ngModel"
+    type="checkbox"
+    ng-required="schema.required">
+<dynamic-content-form-field-input-errors
```

Comment:
The [infographics_spec](https://gist.github.com/benghaziboy/c583e1e56de2d851403e6553ec2f7727) I have has boolean fields, but all with `read_only` and `required` as `false`. I think we might as well implement these validations for this field.